docs/chapters/baseplatform/creating-sdk.rst

```diff
@@ -0,0 +1,14 @@
+Building the SDK installer
+===========================
+
+Building the SDK installer is not very different from creating the corresponding
+target image. All you have to do, is pass an additional option to bitbake: 
+
+.. code-block:: bash
+
+    bitbake -c populate_sdk <image>
+
+This command will create a toolchain installer that contains the sysroot that
+matches your target root filesystem. The installer will be under tmp/deploy/sdk
```

Comment:
The paths would be good to put in code quotation marks. Futhermore, `tmp/deploy/sdk` is somewhat out of context here I believe. Maybe it would be good to point that it is the Yocto tree and the full path is `build/tmp/deploy/sdk`.